### DISCUSSION TITLE
cPanel [Aug 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3216] [Discord Threads]
> 🙋‍♂️ **jackdio**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### u can use it as a nodejs app in cPanel. see https://github.com/nocodb/nocodb-seed  
#  
> 🙋‍♂️ **jackdio**  
#### Thank you for the link, I'll have a read through  
#  
> 🙋‍♂️ **Limesle**  
#### Hi, did you successfully installed it on Cpanel ? I tried nocodb-seed and npm, it seems to install it but I'm struggling to synchronize Url and ports within the "Setup Node.JS App" module of Cpanel.  
#  
> 🙋‍♂️ **Limesle**  
#### jackdio@discord Hi, did you successfully installed it on Cpanel ?  
#  
> 💁 **o1lab**  
#### Which platform are u using?  
#  
> 🙋‍♂️ **Limesle**  
#### Hey ! I'm using a French local provider that uses CPanel  
#  
> 🙋‍♂️ **Limesle**  
#### In this install I tried the module Setup Node.JS App and nocodb-seed  
#  
> 🙋‍♂️ **Limesle**  
#### I fear that I'm facing a CPanel setup issue more than anything to do with nocoDB  
#  
> 🙋‍♂️ **Limesle**  
#### In fact I can't reach the URL as nocoDB is setup for a 8080 local access and CPanel is pushing on a subdomain URL  
#  
> 💁 **o1lab**  
#### Can you see the log?  
#  
> 🙋‍♂️ **Limesle**  
#### On the terminal, after I launch npm run start, I have :    
#### select * from `nc_api_tokens`   
#### Visit : localhost:8080/dashboard  
#  
> 🙋‍♂️ **Limesle**  
#### Can you remind me the name of the log file ?  
#  
> 💁 **o1lab**  
#### There is no logfile. However, it looks good.  
#  
> 💁 **o1lab**  
#### See our environment variables  
#  
> 💁 **o1lab**  
#### PORT  
#  
> 💁 **o1lab**  
#### ~~NC_BASE_URL~~  
#  
> 💁 **o1lab**  
#### If you want to tweak them around  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/getting-started/installation#environment-variables  
#  
> 💁 **o1lab**  
#### sorry NC_PUBLIC_URL  
#  
> 🙋‍♂️ **Limesle**  
#### To be continued tomorrow... I reached my Process usage limit for today making multiple attempts to set the right parameters 😦  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Did you manage to run the nocodb in your cpanel successfully?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/944564926578573313)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
